applications/GPIO/radar_scanner/manifest.yml

```diff
@@ -0,0 +1,15 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/MatthewKuKanich/flipper-radar.git
+    commit_sha: 87faeb83d73a1a8518e5c8dbc8180783a7be3c1c
```

Comment:
You need to update this, also this is what you will be updating via new pull request when you are updating app version

Reply:
Thank you! That makes total sense, updating the hash went over my head but it's how you do versioning. Thanks for letting me know, it's my first app so I'm still learning. It should be set to the latest commit now